tiger_agent/slack.py

```diff
@@ -140,7 +139,8 @@ async def fetch_user_info(client: AsyncWebClient, user_id: str) -> UserInfo | No
         assert isinstance(resp.data, dict)
         assert resp.data["ok"]
         return UserInfo(**(resp.data["user"]))
-    except SlackApiError:
+    except Exception as e:
+        logfire.error("Failed to fetch user info", user_id=user_id, error=str(e))
```

Comment:
Better to use [`logfire.exception`](https://logfire.pydantic.dev/docs/reference/api/logfire/#logfire.Logfire.exception) which will automatically handle capturing the exception, as well as then making sure it's pretty printed within logfire.

```suggestion
        logfire.exception("Failed to fetch user info", user_id=user_id)
```